pkg/sql/copy/testdata/copy_to_binary

```diff
@@ -0,0 +1,18 @@
+exec-ddl
+CREATE TABLE t (id int primary key, t text);
+----
+
+exec-ddl
+INSERT INTO t VALUES
+  (1, 'a tab' || chr(9) || ' separates us'),
+  (2, 'some pipe || characters'),
+  (3, 'new line chars!' || chr(10) || ' ok?'),
+  (4, NULL);
+----
+
+copy-to-error
+COPY t TO STDOUT BINARY
+----
+ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
+HINT: You have attempted to use a feature that is not yet implemented.
+See: https://go.crdb.dev/issue-v/85571/v23.1
```

Comment:
nit: would be nice to make separate issues for BINARY and `file` support.

Reply:
filed:
https://github.com/cockroachdb/cockroach/issues/97180
https://github.com/cockroachdb/cockroach/issues/97181